frontend/src/app/app.routes.ts

```diff
@@ -153,6 +154,10 @@ export const routes: Routes = [
             }
           }
         ]
+      },
+      {
+        path: 'cities',
+        component: CitiesUploadComponent
```

Comment:
It's not, just felt a bit weird to access the page with my non-admin user and couldn't do anything before I realized it was a role issue.